client/src/includes/chooserModal.js

```diff
@@ -290,9 +303,11 @@ class ChooserModalOnloadHandlerFactory {
   }
 
   onLoadReshowCreationFormStep(modal, jsonData) {
-    $(this.creationFormTabSelector, modal.body).replaceWith(
+    $(this.creationFormTabSelector, modal.body).html(
```

Comment:
`html()` sets the inner content of the `creationFormTabSelector` element, while `replaceWith` replaces it. Given that the `data-chooser-creation-form-wrapper` element that `creationFormTabSelector` refers to is the top-level element of creation_form.html - and will thus appear in the htmlFragment - won't this change mean that we end up with two nested `data-chooser-creation-form-wrapper` elements?

Reply:
Hmm yes there is a chance for duplicated nested elements if a creation_form is returned for sure. I may have to revisit this then.

I had originally made this change to avoid having to re-add the same classes / data attributes to all the potential nested templates. For example `confirm_duplicate_upload.html` when using `replaceWith()` will also need to have these attributes/classes for styling/events. 
```
  class="w-chooser-creation-form"
  data-chooser-creation-form-wrapper
```
I will also have to re-update event listeners when the element is replaced if we went with replaceWIth() (not a problem)

I would be interested in knowing all the possible templates that `onLoadReshowCreationFormStep` could potentially send back.